simplesecurity/__init__.py

```diff
@@ -1,212 +1,285 @@
-"""
```

Comment:
Pylint is configured to do a sad if we are missing this. Happy to experiment with ruff so as long as we have this sort of info documented somewhere I'm happy

Reply:
I will propose you something :)

Reply:
I added a configuration into the pyproject.toml and added the dependency.

Reply:
Ok very happy to try ruff :)